release/pipelines/windows-efi-installer/README.md

```diff
@@ -46,16 +46,18 @@ After the ISO is modified it creates a new VirtualMachine which boots from the m
 ```
 
 1. `create-vm-root-disk` creates empty DataVolume which is used for Windows installation.
-2. `import-win-iso` creates new datavolume with Windows ISO file with name defined in `isoDVName` parameter.
+2. `import-win-iso` creates new DataVolume with Windows ISO file with name defined in `isoDVName` parameter. The DataVolume has to be at the same namespace as the PipelineRun (due to the mount of PVC into the PipelineRun pod). 
```

Comment:
```suggestion
2. `import-win-iso` creates new DataVolume with Windows ISO file with name defined in `isoDVName` parameter. The DataVolume needs to be in the same namespace as the PipelineRun (because the PVC is mounted into the PipelineRun pod). 
```